README.md

```diff
@@ -9,6 +9,8 @@ This Gem enables users to develop robotic simulations through ROS2 tools and com
 * Modern version of ROS 2. We support and tested with:
   * [ROS 2 Galactic](https://docs.ros.org/en/galactic/Installation.html) with Ubuntu 20.04.
   * [ROS 2 Humble](https://docs.ros.org/en/humble/Installation.html) with Ubuntu 22.04.
+  * Additional ROS2 packages required:
```

Comment:
This will go to a separate section - see how this is done in vehicle dynamics (easier merge)